Sources/Clappr/Classes/Enum/Event.swift

```diff
@@ -34,4 +33,26 @@ public enum Event: String, CaseIterable {
     case didShowMediaControl
     case willHideMediaControl
     case didHideMediaControl
+    @available(*, deprecated, message: "Update to willSeek")
+    case seek
```

Comment:
Hey @dezouzart , seek was already deprecated before as you can see [here](https://github.com/clappr/clappr-ios/wiki/Events) . Maybe we can remove it.